####2014-10-08 19:15:50
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 271, in login
    proxylist.delProxy()
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 31, in delProxy
    del list[self.point]
TypeError: 'type' object does not support item deletion
####2014-10-08 19:43:10
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 62, in RepostMain
    WeiboMain.login(accountlist=accountlist,proxylist=proxylist,runlogFile=runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 279, in login
    content=urllib2.urlopen('http://weibo.com/1855335174/B96RtlApL?type=repost',timeout=30).read()
  File "E:\Python27\lib\urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "E:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "E:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "E:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "E:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "E:\Python27\lib\urllib2.py", line 1187, in do_open
    r = h.getresponse(buffering=True)
  File "E:\Python27\lib\httplib.py", line 1067, in getresponse
    response.begin()
  File "E:\Python27\lib\httplib.py", line 409, in begin
    version, status, reason = self._read_status()
  File "E:\Python27\lib\httplib.py", line 373, in _read_status
    raise BadStatusLine(line)
BadStatusLine: ''
####2014-10-08 20:22:23
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 62, in RepostMain
    WeiboMain.login(accountlist=accountlist,proxylist=proxylist,runlogFile=runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 271, in login
    proxylist.delProxy()
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 31, in delProxy
    del list[self.point]
TypeError: 'type' object does not support item deletion
####2014-10-08 20:56:56
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 279, in login
    content=urllib2.urlopen('http://weibo.com/1855335174/B96RtlApL?type=repost',timeout=30).read()
  File "E:\Python27\lib\urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "E:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "E:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "E:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "E:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "E:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: <urlopen error [Errno 10060] >
####2014-10-09 14:16:03
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 279, in login
    content=urllib2.urlopen('http://weibo.com/1855335174/B96RtlApL?type=repost',timeout=30).read()
  File "E:\Python27\lib\socket.py", line 351, in read
    data = self._sock.recv(rbufsize)
  File "E:\Python27\lib\httplib.py", line 543, in read
    return self._read_chunked(amt)
  File "E:\Python27\lib\httplib.py", line 597, in _read_chunked
    raise IncompleteRead(''.join(value))
IncompleteRead: IncompleteRead(3076 bytes read)
####2014-10-09 15:54:26
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 149, in RepostMain
    WeiboMain.login(accountlist=accountlist,proxylist=proxylist,runlogFile=runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 279, in login
    content=urllib2.urlopen('http://weibo.com/1855335174/B96RtlApL?type=repost',timeout=30).read()
  File "E:\Python27\lib\urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "E:\Python27\lib\urllib2.py", line 404, in open
    response = self._open(req, data)
  File "E:\Python27\lib\urllib2.py", line 422, in _open
    '_open', req)
  File "E:\Python27\lib\urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "E:\Python27\lib\urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "E:\Python27\lib\urllib2.py", line 1184, in do_open
    raise URLError(err)
URLError: <urlopen error [Errno 10060] >
####2014-10-09 18:46:55
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 263, in login
    weiboMain=WeiboLogin(username,account['pwd'],proxyhttp=proxylist.nextProxy())
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 24, in nextProxy
    return self.list[self.point]
IndexError: list index out of range
####2014-10-09 20:40:01
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 263, in login
    weiboMain=WeiboLogin(username,account['pwd'],proxyhttp=proxylist.nextProxy())
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 24, in nextProxy
    return self.list[self.point]
IndexError: list index out of range
####2014-10-09 21:42:48
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 263, in login
    weiboMain=WeiboLogin(username,account['pwd'],proxyhttp=proxylist.nextProxy())
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 24, in nextProxy
    return self.list[self.point]
IndexError: list index out of range
####2014-10-10 18:34:04
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 263, in login
    weiboMain=WeiboLogin(username,account['pwd'],proxyhttp=proxylist.nextProxy())
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 24, in nextProxy
    return self.list[self.point]
IndexError: list index out of range
####2014-10-10 21:57:26
Traceback (most recent call last):
  File "E:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    RepostMain.RepostMain(accountlist,line[:-1],checkey,figure,proxylist)
  File "E:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 263, in login
    weiboMain=WeiboLogin(username,account['pwd'],proxyhttp=proxylist.nextProxy())
  File "E:\MyProjects\python\weiboCrawl\WeiboMain.py", line 24, in nextProxy
    return self.list[self.point]
IndexError: list index out of range
####2014-10-12 12:15:19
Traceback (most recent call last):
  File "e:\MyProjects\python\weiboCrawl\main.py", line 53, in <module>
    if line[-1]=='\n':
  File "e:\MyProjects\python\weiboCrawl\RepostMain.py", line 109, in RepostMain
    get_content_result=WeiboMain.get_content(url,runlogFile,accountlist,proxylist)
  File "e:\MyProjects\python\weiboCrawl\WeiboMain.py", line 223, in get_content
    login(accountlist,proxylist,runlogFile)
  File "e:\MyProjects\python\weiboCrawl\WeiboMain.py", line 263, in login
    weiboMain=WeiboLogin(username,account['pwd'],proxyhttp=proxylist.nextProxy())
  File "e:\MyProjects\python\weiboCrawl\WeiboMain.py", line 24, in nextProxy
    return self.list[self.point]
IndexError: list index out of range
